docker/images/uptime-tracker/Dockerfile

```diff
@@ -1,32 +1,46 @@
 ARG image_tag
 ARG base_image
 
-FROM ${base_image} as builder
+FROM ${base_image} as uptime-tracker-build
```

Comment:
Are we keeping the uptime tracker image and Dockerfile in this repo to avoid code duplication? I would prefer moving this over to the Uptime tracker repo so we have a clear and easy setup for each repo. If we try to build the uptime tracker here, users who want to use our make targets are going to fail at this point.

Reply:
I see that we remove the uptime tracker image from the `docker-push.sh` script. So we should move this whole code related to building the uptime tracker to the uptime tracker repo.

Reply:
We use uptime-tracker images (dockerfile) here for integration env, same as before we used for skycoin-service-discovery and dmsg and skywire. And those files we move to ut repo itself used for deploying stage on github-action.